07-advanced-git/README.md

```diff
@@ -1 +1,6 @@
 # WIP
+
+Sections:  
+
+7. Advanced Git
+   1. `Git Log`
```

Comment:
```suggestion
1. [`Git Log`](./07-01-git-log/README.md)
```
Will need to test to ensure that the path is right

Reply:
I think this table of contents will probably change in the future but I guess we can keep it like this for now